01-html/04-nytimes.html

```diff
@@ -1,3 +1,1120 @@
+          <li><a href="https://myaccount.nytimes.com/">Manage Account</a></li>
+          <li><a href="http://www.nytimes.com/pages/todayspaper/index.html">Today's Paper</a></li>
+          <li><a href="http://www.nytimes.com/interactive/blogs/directory.html">Blogs</a></li>
+          <li><a href="http://www.nytimes.com/pages/topics/">Times Topics</a></li>
+          <li><a href="http://www.nytimes.com/marketing/tools-and-services/">Tools & Services</a></li>
+          <li><a href="http://www.nytimes.com/section/jobs">Jobs</a></li>
+          <li><a href="http://www.nytimes.com/ref/classifieds/">Classifieds</a></li>
+          <li><a href="http://www.nytimes.com/pages/corrections/index.html">Corrections</a></li>
+        </ul>
+    </nav>
+    <div id="search" class="inline-block">
+      <img src="http://uploads.webflow.com/55cf187a8cacc9ab28d524b9/55d243680a3ebde1455ffab4_transparency.png" height="30" alt="search-icon">SEARCH</div>
+    <button class="inline-block"><a href="http://www.nytimes.com/subscriptions/Multiproduct/lp3004.html?module=Subscribe-Now&pgtype=Homepage">SUBSCRIBE NOW</a></button>
+    <button class="inline-block"><a href="http://www.nytimes.com/?action=Click&region=TopBar&module=EditionToggleToUS&pgtype=Homepage&WT.z_jog=1&hF=t&vS=undefined">LOG IN</a></button>
+    <nav id="gear-menu">
+      <img src="http://simpleicon.com/wp-content/uploads/gear-21.svg" alt="menu-gear" height="20">
+      <ul class="no-text-decoration">EDITION
+        <li><a href="http://www.nytimes.com/section/us">U.S.</a></li>
+        <li><a href="http://international.nytimes.com/?WT.z_jog=1&hF=t&vS=undefined">International</a></li>
+        <li><a href="http://cn.nytimes.com/zh-hant/"><img src="http://virtusjustitia.vjcdn.com/wp-content/uploads//2014/04/20121025032309-89444.jpg" height="15" alt="mallet-stickfigure">(Chinese)</a>
+        </li>
+      </ul>
+      <ul class="no-text-decoration">HELP
+        <li><a href="http://www.nytimes.com/content/help/front.html">FAQ</a></li>
+        <li><a href="http://www.nytimes.com/content/help/contact/directory.html">Contact Us</a></li>
+      </ul>
+    </nav>
+    <h1>
+      <a href="http://www.nytimes.com/">
+        <img src="http://www.newspapers.psu.edu/wp-content/uploads/sites/1856/2013/02/NYTLogo.jpg" alt="nyt-logo" height="100">
+      </a>
+    </h1>
+    <nav id="day-info" class="inline-block">
+      <ul>
+        <li>Friday, January 29, 2016</li>
+        <li><a href="http://www.nytimes.com/pages/todayspaper/index.html">Today's Paper</a></li>
+        <li>
+          <a href="http://www.nytimes.com/video">
+            <img src="http://www.clipartbest.com/cliparts/niB/Bgg/niBBggBLT.png" alt="camera" height="10">Video</a>
+        </li>
+        <li class="onmouseover">
+          <img src="https://www.kelownanow.com/img/weather-icons/mcloudy.png" height="20" alt="weather">48&#8457;</li>
+        <li id="stocks" class="onmouseover">Wall Street API</li>
+      </ul>
+    </nav>
+    <hr>
```

Comment:
You're almost always better off displaying a horizontal line as a border on an element instead of an `hr`. If it's not content or doesn't contain content, it probably doesn't belong in the HTML.